ztp/source-crs/du-profile-kube-compare/required/node-tuning-operator/PerformanceProfile.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  # if you change this name make sure the 'include' line in TunedPerformancePatch.yaml
+  # matches this name: include=openshift-node-performance-${PerformanceProfile.metadata.name}
+  # Also in file 'validatorCRs/informDuValidator.yaml': 
+  # name: 50-performance-${PerformanceProfile.metadata.name}
+  name: openshift-node-performance-profile
+  annotations:
+    ran.openshift.io/reference-configuration: "ran-du.redhat.com"
+spec:
+  additionalKernelArgs:
+  - "rcupdate.rcu_normal_after_boot=0"
+  - "efi=runtime"
```

Comment:
In previous versions of the reference  "efi=runtime"`  was optional was checked by:
```
  {{- $containsEFI := false }}{{- range .spec.additionalKernelArgs }}{{ if eq . "efi=runtime" }}{{ $containsEFI = true }}{{ end }}{{ end }}{{ if eq $containsEFI true }}
  - "efi=runtime"
  {{- end }}
  ~
  ```
  not sure what is the expected behavior just wanted to mention that its different

Reply:
Will double check!